src/com/ableton/Pyenv.groovy

```diff
@@ -45,21 +45,22 @@ class Pyenv implements Serializable {
 
     VirtualEnv venv = new VirtualEnv(script, randomSeed)
     int result
-    List installCommands = [
-      "export PYENV_ROOT=${pyenvRoot}",
-      "export PATH=\$PYENV_ROOT/bin:\$PATH",
-      "eval \"\$(pyenv init --path)\"",
-      "eval \"\$(pyenv init -)\"",
-      "pyenv install --skip-existing ${pythonVersion}",
-      "pyenv shell ${pythonVersion}",
```

Comment:
pyenv-win supports `pyenv shell`, so I'm not sure why we can't use it on Windows.

Reply:
IIRC, when I last tested this, it didn't work for us. I'll try it again, though.

Reply:
I just pushed a WIP commit to test this, and indeed, it failed. So while it is supported, it doesn't seem to work from Jenkins for the shell session.

Reply:
That's a shame... Well, I guess setting the `PYTHON_VERSION` env var is not that bad.